core/src/components/modal/gestures/swipe-to-close.ts

```diff
@@ -24,11 +24,11 @@ export const createSwipeToCloseGesture = (
       return true;
     }
 
-    const content = target.closest('ion-content');
-    if (content === null) {
+    const contentOrFooter = target.closest('ion-content, ion-footer');
+    if (contentOrFooter === null) {
       return true;
     }
-    // Target is in the content so we don't start the gesture.
+    // Target is in the content or the footer so we don't start the gesture.
```

Comment:
```suggestion
    // Target is in the content or the footer so do not start the gesture.
```

GitHub Actions had an outage yesterday, so I need to add a new commit that will get the actions to fire again ;)